desktop/core/src/desktop/lib/botserver/views.py

```diff
@@ -134,20 +130,31 @@ def query_result(request, notebook):
   history_uuid = query_execute['history_uuid']
   status = _check_status(request, operation_id=history_uuid)
   if status['query_status']['status'] == 'available':
-    response = _fetch_result_data(request, operation_id=history_uuid)
-    return response['result']
+    res = _fetch_result_data(request, operation_id=history_uuid, rows=max_rows)
+
+    # Make result pivot table
```

Comment:
Not sure why in `query_result` and not `_make_result_table`?

PS: also query_result no leading _?

Reply:
Initially kept in `query_result` because of https://github.com/cloudera/hue/pull/1901#discussion_r595226101 logic
But since we are not truncating the result and keeping it as it is, so its better to move in `_make_result_table` function @romainr 

Also adding the leading _ in query_result function